_docs/reference/api/reference.md

```diff
@@ -0,0 +1,15 @@
+---
+title: API Reference
+
+overview: An overview pointing at api reference docs
+
+order: 42
```

Comment:
What is this?

Reply:
its just stuff from the template to get stuff magically indexed.

Reply:
fixed.